client/src/containers/BBLPage.tsx

```diff
@@ -64,7 +64,7 @@ const BBLPage: React.FC<BBLPageProps> = (props) => {
               ...results.result[0],
               locale,
             },
-            props.useNewPortfolioMethod
+            !props.useNewPortfolioMethod
```

Comment:
so ideally we'd refactor this too to be useOldPortfolioMethod but i get it if you wanna just get this in